.circleci/config.yml

```diff
@@ -427,13 +443,15 @@ jobs:
     steps:
       - checkout
       - *attach-to-workspace
+      - *restore-assets-cache
       - run:
           name: Precompile assets
           command: |
             cp config/examples/*.yml config/
-            bundle exec rake assets:precompile NODE_ENV=test RAILS_ENV=test
+            bundle exec rake assets:precompile assets:clean NODE_ENV=test RAILS_ENV=test
```

Comment:
clean step doesn't add a significant amount of time, less than a second as far as I see locally. It should help avoid cache bloat over time and is needed because of line 70.